### PR TITLE
Rw refactor store gps

### PR DESCRIPTION
**Pull request by rufusasterisk**
## What's this PR do?

This PR merges 3 store variables (yourGPSLocation, friendGPSLocation, mainGPSLocation) into a single store object gpsLocations with keys of your, friend, and main containing the GPS objects.

## Pre-Merge TODOs
 - [X] Tests Added
   - ANY change that can be tested must include testing
 - [-] Updated Documentation
   - Update your README if any new dependencies are added 
 - [-] Add follow up issues?
 - [x] 2 PR reviews completed

## Learning

No new material was required for the refactor.

## Where should the reviewer start?

EndpointSearchReducers line #172 - this is the new reducer to capture the actions. The code removed from EndpointSearchReducers are the old reducers. The name changes carry through the rest of the files for the refactor

## Any background context you want to provide?

I had three reducers doing almost the same thing - the action is concise, but storing the data required almost the same function in triplicate.

### CODE REVIEW

**Review by lfinney — APPROVED** [review]
Solid refactor into the one reducer

File: src/reducers/EndpointSearchReducers.js [hunk 1]
```diff
@@ -78,20 +64,6 @@ export const friendTextLocation = (state = '', action) => {
   }
 };
 
-// export const friendGPSLocation = (state = {lat: 0, lng: 0}, action) => {
```
Comment (nicktu12) [inline src/reducers/EndpointSearchReducers.js]: Remove all the dead code. Great workflow and atomic commits!

**Review by lolakoala — APPROVED** [review]
This refactor looks clean and concise. Great work! Looks like it makes your code a good bit cleaner.

**Review by nicktu12 — APPROVED** [review]
Well structured code makes it easy to update tests! I'm excited to see this in action.

**Comment by rufusasterisk** [comment]
Thanks for all the reviews!

**Pull request merged**